### PR TITLE
Enable basic SSL support for non-auth connections

### LINKED ISSUE #84: Basic SSL support
Is it possible to attach to a kafka instance with SSL enabled?  For example, when I'm attaching to my cluster as a producer, I'll use the following:

KafkaProducer(bootstrap_servers=kafkaServer, security_protocol="SSL", ssl_check_hostname=True)

When attaching to the kafka cluster through this extension, I can specify the bootstrap server and port (443 in my case), but I can't seem to find a way to change the security protocol.

### PR DESCRIPTION
**Pull request by fbricon**
also fixes the total steps value, depending on the selected auth option

Screenshot 2021-01-19 at 16 56 05

Fixes #84

### CODE REVIEW

**Comment by fbricon** [comment]
@mike-england: please unzip and manually install this [vsix](https://github.com/jlandersen/vscode-kafka/suites/1858081264/artifacts/36088011), and check whether creating a non-auth'ed SSL-enabled cluster works for you

File: src/wizards/clusters.ts [hunk 1]
```diff
@@ -12,12 +12,18 @@ interface AddClusterState extends State {
     bootstrap: string;
     name: string;
     saslOption: SaslOption;
+    ssl: boolean;
```
Comment (angelozerr) [inline src/wizards/clusters.ts:15]: What about extending ConnectionOptions  ?

File: src/wizards/clusters.ts [hunk 2]
```diff
@@ -70,9 +76,15 @@ export async function addClusterWizard(clusterSettings: ClusterSettings, explore
...
         })).label;
-        if (authentification && authentification != authOptions[0].label) {
-            state.saslOption = { mechanism:  authMechanisms.get(authentification) as SaslMechanism};
-            return (input: MultiStepInput) => inputAuthentificationUserName(input, state);
+        if (authentification ) {
+            if (authentification == authOptions[0].label) {
+                state.totalSteps = DEFAULT_STEPS;// we're on the 4-step track
+                return (input: MultiStepInput) => inputSSL(input, state);
+            } else {
+                state.totalSteps = DEFAULT_STEPS+1;// we're on the 5-step track
```
Comment (angelozerr) [inline src/wizards/clusters.ts:84]: Please format the file.

File: src/wizards/clusters.ts (hunk 1 above)
Comment (fbricon) [reply src/wizards/clusters.ts:15]: it already extends State

File: src/wizards/clusters.ts (hunk 2 above)
Comment (fbricon) [reply src/wizards/clusters.ts]: done

File: src/wizards/clusters.ts (hunk 1 above)
Comment (angelozerr) [reply src/wizards/clusters.ts:15]: I mean:

```ts
interface AddClusterState extends State, ConnectionOptions {
    name: string;
}
```

Comment (fbricon) [reply src/wizards/clusters.ts]: done

**Comment by angelozerr** [comment]
Rhanks @fbricon !

**Pull request merged**